README.md

```diff
@@ -61,10 +69,11 @@ Either from QGIS plugin repository or manually:
 
 ## License
 
-This project is published under the GPLv3 license, see [LICENSE.md](https://github.com/GIScience/orstools-qgis-plugin/blob/master/LICENSE.md) for details.
+This project is published under the GPLv3 license, see [LICENSE.md](https://github.com/GIScience/orstools-qgis-plugin/blob/main/LICENSE.md) for details.
 
-By using this plugin, you also agree to the terms and conditions of openrouteservice, as outlined [here](https://openrouteservice.org/terms-of-service/).
+By using this plugin, you also agree to the [terms and conditions](https://openrouteservice.org/terms-of-service/) of
+openrouteservice.
 
 ## Acknowledgements
 
-This project was first started by [Nils Nolde](https://github.com/nilsnolde), [GIS • OPS UG](https://gis-ops.com).
+This project was first started by [Nils Nolde](https://github.com/nilsnolde).
```

Comment:
Is there a reason for this deletion?

Reply:
yes, we should not (are not allowed to) promote external companies. Name of the author should be sufficient.

Reply:
:+1: